docs/en/25-application/02-collectd.md

```diff
@@ -37,7 +37,7 @@ Please refer to the [official documentation](https://grafana.com/grafana/downloa
 
 ### Install TDengine
 
-Download the latest TDengine-server 2.4.0.x or above from the [Downloads](http://taosdata.com/cn/all-downloads/) page on the TAOSData website and install it.
+Download the latest TDengine-server from the [Downloads](http://taosdata.com/cn/all-downloads/) page on the TAOSData website and install it.
```

Comment:
Use tdengine.com download url in English docs?
But https://tdengine.com/all-downloads/ is still 2.6 versions.

@gccgdb1234